docs/styles.css

```diff
@@ -158,7 +158,7 @@ body {
 
   .navbar__links {
     text-align: center;
-    padding: 1rem;
+    padding: 1rem 0;

```

Comment:
Alternatively, if you still want padding on the left, you could do:

```
padding: 1rem 0 1rem 1rem;
```

to set all but right to 1rem